py/torch_tensorrt/fx/passes/pass_utils.py

```diff
@@ -179,6 +179,27 @@ def bounded_method(*args, **kwargs):
     return dec_for_method
 
 
+def req_torch_version(min_torch_version: str = "2.0") -> Decorator:
```

Comment:
Lets try to use some sort of semver style versioning here. String might not be the best type